src/tags/TagActions.js

```diff
@@ -55,11 +55,15 @@ const allTagsRequestError = err => ({ type: ALL_TAGS_REQUEST_ERROR, err });
 
 export const getTags = () => (dispatch, getState) => {
   dispatch(tagRequestStarted());
-  fetch(getState().app.pillarHost + API_PREFIX + 'tags', getInit(null, 'GET'))
+  return fetch(getState().app.pillarHost + API_PREFIX + 'tags', getInit(null, 'GET'))
     .then(
-      response => response.json()
+      response => {
+        return response.ok ? response.json() : Promise.reject(response.status + ' ' + response.statusText);
```

Comment:
why is this here, instead of in the `catch`? I thought that if the response has an error code > 399 that fetch rejects the promise